.dockerignore

```diff
@@ -10,5 +10,5 @@ jspm_packages/
 infrastructure/
 monitoring/
 ci/
-test/
+#test/
```

Comment:
Is this change deliberate?

Reply:
yes, need to include test files in image